igvc_gazebo/launch/igvc_control.launch

```diff
@@ -9,7 +9,17 @@
   <node name="controller_spawner" pkg="controller_manager" type="spawner"
     args="$(find igvc_gazebo)/config/igvc_joint_control.yaml"/>
 
-  <node name="igvc_control" pkg="igvc_gazebo" type="control">
+  <node name="igvc_control" pkg="igvc_gazebo" type="control" output="screen">
+      <!-- PID values for wheel speed -->
+      <param name="speed_P_left" value="0.2"/>
```

Comment:
These really low values might explain why the robot takes a long time to converge to the path in simulation

Reply:
It ramps up equivalent to what I saw with Oswin's step inputs. You can change the PID values though. Separate PR IMO.